### PR TITLE
build: binary for armv6 gnueabihf

### PR DESCRIPTION
**Pull request by xabi**
hi ! it would be great to use candlex for  small machines where the default nx backend is very very slow. I've been able to easily compile candlex for this target

### CODE REVIEW

File: lib/candlex/native.ex [hunk 1]
```diff
@@ -15,6 +15,7 @@ defmodule Candlex.Native do
     version: version,
     nif_versions: ["2.16"],
     targets: [
+      "arm-unknown-linux-gnueabihf",
       "aarch64-apple-darwin",
       "aarch64-unknown-linux-gnu",
       "x86_64-apple-darwin",
```
Comment (grzuy) [inline lib/candlex/native.ex:21]: Can we please keep alphabetical order here?

**Review by grzuy — APPROVED** [review]
Hi @xabi,

Thanks for your contribution.

Happy to merge soon and release 0.1.5 so you can test the new binary.
Interested in the results you get later on if you want to report back :-)

File: lib/candlex/native.ex (hunk 1 above)
Comment (grzuy) [reply lib/candlex/native.ex:24]: Updated.

**Comment by grzuy** [comment]
`v0.1.5` published (https://hex.pm/packages/candlex/0.1.5).

**Comment by xabi** [comment]
Thx for the merge :)

**Pull request merged**